sample_factory_examples/atari_examples/enjoy_atari.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+from sample_factory.cfg.arguments import parse_args
+from sample_factory.enjoy import enjoy
+from sample_factory_examples.atari_examples.train_atari import register_atari_components
+
+
+def main():
+    """Script entry point."""
+    register_atari_components()
```

Comment:
To add to the point above, here you can for example add `evaluation=True` parameter to register_atari_components and use this argument to set the render mode if this is really needed.
If this is needed only on Mac you can even check for which OS you're using

Reply:
https://stackoverflow.com/questions/8220108/how-do-i-check-the-operating-system-in-python

Reply:
Sounds like a good way.